docs/content/management-introduction.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Overview & Architecture"
+kind: management
+weight: 1
+---
+
+OPA exposes a set of APIs that enable unified, logically centralized policy
+management. Read this page if you are interested in how to build a control plane
+around OPA that enables policy distribution and collection of important
+telemetry data like decision logs.
+
+## Overview
```

Comment:
Nit: should we remove this header? I feel like the ToC looks a bit odd:

![image](https://user-images.githubusercontent.com/238885/115555600-ae12a000-a27d-11eb-87c8-604a6f4cf3cc.png)

Reply:
Oh, yeah it absolutely does! I'll remove that, thanks!